Sources/BraintreePayPal/BTPayPalClient.swift

```diff
@@ -436,7 +436,7 @@ import BraintreeDataCollector
                         )
                         return
                     }
-                    let merchantAccountID = configuration.merchantID
+                    let merchantAccountID = json["merchantId"].asString()
```

Comment:
Should we rename this property and the `launchPayPalApp` property to `merchantID` to properly represent the value?